### PR TITLE
Mar/ipld pathing

### PR DESCRIPTION
**Pull request by mrodriguez3313**
- [x]  Created write up tutorial
- [x] Embeded/Created submission form with answer verification (using regex)
- [x]  Completed video editing and re-record some parts for IPFS/Kubo calrity.

### CODE REVIEW

**Comment by walkerlj0** [comment]
This doesn't need to be added to Learning Content board, you can just put it in this ticket: https://github.com/orgs/protocol/projects/21/views/2?pane=issue&itemId=4453123

**Review by walkerlj0 — APPROVED** [review]
Great work with the video!

File: content/en/tutorials/ipld/ipld-pathing.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+## Instructions
+
+### Install kubo (go-ipfs)
+
+Refer to the [IPFS Docs’ installation instructions](https://docs.ipfs.tech/install/command-line) for the operating system you are using.
+
+### Initialize IPFS Node
+
+Before you can run the ipfs daemon, you have to initialize an instance of IPFS on your machine, this will. After which you can open a new terminal to continue on with the tutorial.
```
[the remark this answers is not in the document]
Comment (mrodriguez3313) [reply content/en/tutorials/ipld/ipld-pathing.md:43]: Actually its strange that those instructions are there in the first place, bc your first PR fixed this

[the remark this answers is not in the document]
Comment (mrodriguez3313) [reply content/en/tutorials/ipld/ipld-pathing.md:43]: This file was supposed to be deleted

**Pull request merged**